tests/platform_tests/test_idle_driver.py

```diff
@@ -0,0 +1,25 @@
+"""
+Some devices have potential problems entering idle state. So
+on m0 devices, we expect to disable both intel idle driver
+and acpi idle driver, and no available idle state for all cpu.
+"""
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('m0'),
```

Comment:
Should we enable this testcase on `mx` and other topology?

Reply:
Echo on this. We need enable for Mx as well. Please update the header as well